homeassistant/components/fronius/quality_scale.yaml

```diff
@@ -0,0 +1,87 @@
+rules:
```

Comment:
And these
```py
    FroniusSensorEntityDescription(
        key="led_state",
        entity_category=EntityCategory.DIAGNOSTIC,
        entity_registry_enabled_default=False,
    ),
    FroniusSensorEntityDescription(
        key="led_color",
        entity_category=EntityCategory.DIAGNOSTIC,
        entity_registry_enabled_default=False,
    ),
```

Reply:
There's a LED on the device. This shows some integers representing state and color. No idea what this could be used for - or what color "2" is 😬

Reply:
So if you do not know, how is the user supposed to know?

Reply:
That's why it's disabled by default. Should the entity be removed completely?

Reply:
I would either make a way to complete the list, as in, maybe just check if the color is in a list, and if it is not, return `None` as value and log. After you collected a bunch of them you can remove that and it will be a bit more unforgiving, but then you at least collected 90% of all options